src/main/java/org/springframework/data/redis/core/RedisOperations.java

```diff
@@ -269,24 +269,24 @@ <T> T execute(RedisScript<T> script, RedisSerializer<?> argsSerializer, RedisSer
 	K randomKey();
 
 	/**
-	 * Rename key {@code oldKey} to {@code newKey}.
+	 * Rename key {@code key} to {@code newKey}.
 	 *
-	 * @param oldKey must not be {@literal null}.
+	 * @param key must not be {@literal null}.
 	 * @param newKey must not be {@literal null}.
 	 * @see <a href="https://redis.io/commands/rename">Redis Documentation: RENAME</a>
 	 */
-	void rename(K oldKey, K newKey);
+	void rename(K key, K newKey);
 
 	/**
-	 * Rename key {@code oldKey} to {@code newKey} only if {@code newKey} does not exist.
+	 * Rename {@code key} to {@code newKey} only if {@code newKey} does not exist.
 	 *
-	 * @param oldKey must not be {@literal null}.
+	 * @param key must not be {@literal null}.
 	 * @param newKey must not be {@literal null}.
 	 * @return {@literal null} when used in pipeline / transaction.
 	 * @see <a href="https://redis.io/commands/renamenx">Redis Documentation: RENAMENX</a>
 	 */
 	@Nullable
-	Boolean renameIfAbsent(K oldKey, K newKey);
+	Boolean renameIfAbsent(K key, K newKey);
```

Comment:
I think the renames are fine.

But, I think the Javadoc needs work. For example, what does the `oldKey`/`key` refer to? Is a key to a data structure in Redis? Is it a key in a hash data structure. I know I was confused by this when I first started learning about Redis.

Reply:
It is not our responsibility to educate folks about the concepts of their data store.